src/pages/GamePage.js

```diff
@@ -19,10 +21,19 @@ class GamePage extends Component {
                 <div id="question-box">
                     <Question build={inputs}/>
                     <p id="question-left"><span id="question-left-count">5</span> questions left</p>
+                    <p>Connected to server: { this.props.gameServerLink }</p>
+                    <p>Using nickname: { this.props.nickname }</p>
```

Comment:
These two lines are just for debug purposes.
I did not remove them as they might be useful for later tickets.